clang-tools-extra/clang-tidy/utils/UseRangesCheck.cpp

```diff
@@ -164,6 +165,24 @@ void UseRangesCheck::registerMatchers(MatchFinder *Finder) {
 static void removeFunctionArgs(DiagnosticBuilder &Diag, const CallExpr &Call,
                                ArrayRef<unsigned> Indexes,
                                const ASTContext &Ctx) {
+  auto GetCommaLoc = [&](SourceLocation BeginLoc,
+                         SourceLocation EndLoc) -> CharSourceRange {
+    SourceLocation CommaLoc =
```

Comment:
```suggestion
    const SourceLocation CommaLoc =
```

Same for other variables below.